update with new thorin

storywithoutend opened the pull request.

TateB commented:
small notes from visual review:
- hamburger button not aligned with ENS logo
- profile dropdown doesn't look great with border, also looks too small
- disabled "Favourites" button doesn't look disabled
- language dropdown has left aligned text that looks awkward
- back button doesn't have padding so the hover state is odd
- we should disable the minus button for the expiry selector in registration (it was like that before but only just noticed it)
- these lines should be lighter (in registration):
<img width="684" alt="Screenshot 2023-01-19 at 4 30 11 pm" src="https://user-images.githubusercontent.com/11844316/213363031-7a1e32f6-9273-4151-a52c-9b22f17b30f9.png">
- these icons don't look good centre aligned:
<img width="423" alt="Screenshot 2023-01-19 at 4 32 21 pm" src="https://user-images.githubusercontent.com/11844316/213363270-067c6c19-e404-461e-91eb-a5df0243dd77.png">
- the transition on the add account button is awkward, the border pops in once the animation finishes
- the clear button doesn't have a hover state in inputs (or a pointer cursor)

storywithoutend commented:
> small notes from visual review:
> 
> * hamburger button not aligned with ENS logo

Couldn't get find the situation where it would be offline.

> * profile dropdown doesn't look great with border, also looks too small



> * disabled "Favourites" button doesn't look disabled

Updated header icons and links to have hover state and disabled state similar to the Profile tabs.

> * language dropdown has left aligned text that looks awkward

I increased the size of the button and switched to alignment to centered.

> * back button doesn't have padding so the hover state is odd

Added padding

> * we should disable the minus button for the expiry selector in registration (it was like that before but only just noticed it)

Fixed

> * these lines should be lighter (in registration):
> 
> <img alt="Screenshot 2023-01-19 at 4 30 11 pm" width="684" src="https://user-images.githubusercontent.com/11844316/213363031-7a1e32f6-9273-4151-a52c-9b22f17b30f9.png">
> 

Didn't fix it since the registration profile editor is going to change.

> - these icons don't look good centre aligned:
> <img alt="Screenshot 2023-01-19 at 4 32 21 pm" width="423" src="https://user-images.githubusercontent.com/11844316/213363270-067c6c19-e404-461e-91eb-a5df0243dd77.png">
> 

Didn't fix since permissions menu will be changing.

> - the transition on the add account button is awkward, the border pops in once the animation finishes - the clear button doesn't have a hover state in inputs (or a pointer cursor)

Fixed. Removed the border of the button and switched to using border on outer container.

storywithoutend commented on src/components/@molecules/AddRecordButton/AddRecordButton.tsx:
```diff
@@ -251,11 +214,17 @@ const NoOptionsContainer = styled.div<{ $inline: boolean }>(
 
+const SVGWrapper = styled.svg(
+  ({ theme }) => css`
+    color: ${theme.colors.greyPrimary} !important;
```
Switched to a wrapper div.
